learningobjects/models.py

```diff
@@ -32,12 +32,12 @@ class LearningObject(models.Model):
     path_xml = models.TextField()
     mpath = models.TextField()
```

Comment:
This may be the wrong place to mention this but are we renaming this to materialized_path? Or using MPTT per @carsongee suggestion? I have no opinion either way, just making sure it wasn't accidentally overlooked